.github/workflows/publish.yml

```diff
@@ -13,17 +15,70 @@ jobs:
 
       - name: Set up Quarto
         uses: quarto-dev/quarto-actions/setup@v2
+
+      - name: Set up R
+        uses: r-lib/actions/setup-r@v2
+
+      - uses: r-lib/actions/setup-r-dependencies@v2
+        with:
+          pak-version: devel
+
+      - name: Render Quarto Project
+        uses: quarto-dev/quarto-actions/render@v2
+
+      - name: Restore lychee cache
+        uses: actions/cache@v3
         with:
-          # To install LaTeX to build PDF book
-          tinytex: true
-          # uncomment below and fill to pin a version
-          # version: SPECIFIC-QUARTO-VERSION-HERE
+          path: .lycheecache
+          key: cache-lychee-${{ github.sha }}
+          restore-keys: cache-lychee-
 
-      # add software dependencies here
+      - name: Link Checker
+        id: lychee
+        uses: lycheeverse/lychee-action@v1
+        with:
+          fail: true
+          args: "docs --exclude-loopback --insecure --exclude-mail --exclude-path docs/site_libs --cache --max-cache-age 1d"
```

Comment:
The link https://community.jitsi.org/t/how-can-i-add-extra-jvb-in-docker-meet/30911/36 works in the browser but not with this. What's the standard process in cynkraweb for dealing with this? Should we add some headers?

Reply:
In cynkraweb, this link was added to `.lycheeignore`. Do we have something similar? A cruder solution could be to simply remove the link?